pipeline_utils/__main__.py

```diff
@@ -20,8 +20,8 @@
 
 # Variables
 PIPELINE_DEPLOY = 'pipeline_deploy'
-CGAP_ALIAS = 'cgap-core'
-DBMI_ALIAS = 'hms-dbmi'
+CONSORTIA_ALIAS = 'CONSORTIUM'
+SUBMISSION_CENTERS_ALIAS = 'SUBMISSION_CENTER'
```

Comment:
For deploying pipelines odds are we will want these to be `smaht-consortia` viewable with no submission center ie: only available for add/edit by admins. In some select cases we may allow submission centers to POST these though if they do end up contributing back.

Reply:
yea, this is just a placeholder I will replace with the right one as soon as we have a decision